src/wobblySpinnerSheet.js

```diff
@@ -0,0 +1,72 @@
+import * as keyframesSheet from "./keyframesSheet";
+import wobblySpinnerRules from "./wobblySpinnerRules";
+import jss from "jss";
+import nested from "jss-nested";
+
+jss.use(nested()); // Allow for `&::after` in rules.
+
+const sheetRefCountMap = new Map(); // Map of JSS created `sheet` and `refCount`, indexed by `rules`.
+
+function attachSheet(sheet, rules) {
+    if (!sheet) {
+        sheet = jss.createStyleSheet(rules, {meta: "WobblySpinner"});
+    }
+    sheet.attach();
+    return sheet;
+}
+
+function detachSheet(sheet) {
+    sheet.detach();
+}
+
+function createRulesFromProps({diameter, thickness, wobbleOffset, spinRate, outerColor, innerColor, wobbleColor}) {
+    return wobblySpinnerRules({
+        thickness,
+        afterTopLeft: -thickness + wobbleOffset,
+        diameter,
+        spinRate,
+        wobbleRate: spinRate * 1.5,
+        borderRadius: (diameter + (thickness * 2)) / 2,
```

Comment:
Why aren't these calculations part of spinner rules? Does this module just manage the lifecycle of the sheets?